helm/squidex/values.yaml

```diff
@@ -46,6 +46,12 @@ affinity: { }
 ## @skip clusterSuffix
 clusterSuffix: cluster.local
 
+## @skip podSecurityContext
+podSecurityContext: { }
```

Comment:
An example of a **Pod Security Context** that follows the rules defined by the strictest policy outlined by Kubernetes **Restricted**:

```
podSecurityContext:
  seccompProfile:
    type: RuntimeDefault
  runAsNonRoot: true
  runAsUser: 10000
  runAsGroup: 10000
  fsGroup: 10000
```